string-inflection.el

```diff
@@ -117,6 +126,15 @@ If include `:', select `FOO::VERSION' to run `M-x\ string-inflection-underscore'
 
 (fset 'string-inflection-cycle 'string-inflection-ruby-style-cycle)
 
+;;;###autoload
+(defun string-inflection-elixir-style-cycle ()
+  "foo_bar => FooBar => foo_bar"
+  (interactive)
+  (string-inflection-insert
+   (string-inflection-elixir-style-cycle-function (string-inflection-get-current-word))))
+
+(fset 'string-inflection-cycle 'string-inflection-elixir-style-cycle)
```

Comment:
Overwriting the same definition. (Line 127)
If necessary, do it outside the library

Reply:
Ah, apologies; accidental copy/paste duplication. Removed.